bcgov/moh-prime
# PRIME-2061 Licence refactor with details

To test:

- verify lookup returns the same data 
- work through the enrollee workflow and add a college certificate
- verify there are no errors in the rules engine and the proper automatic or manual status is achieved.
